csrc/host_ir/host_ir.cpp

```diff
@@ -119,6 +119,43 @@ bool PostOnStream::sameAs(const Statement* other) const {
   return false;
 }
 
+LaunchKernel::LaunchKernel(
+    IrBuilderPasskey passkey,
+    int hic_executor_index,
+    std::vector<Val*> inputs,
+    std::vector<Val*> outputs)
+    : Expr(passkey, std::move(inputs), std::move(outputs), {}),
+      hic_executor_index_(hic_executor_index) {}
+
+NVFUSER_DEFINE_CLONE_AND_CREATE(LaunchKernel)
+
+std::string LaunchKernel::toString(int indent_size) const {
+  std::stringstream ss;
+  indent(ss, indent_size) << "LaunchKernel ("
```

Comment:
```suggestion
  indent(ss, indent_size) << "LaunchKernel("
```
Nit: a style convention